### PR TITLE
[googletts] Configuration example via text file in documentation

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.voice.googletts/README.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ It is recommended to clear this configuration parameter afterwards.
 When enabled the cache is purged once.
 Make sure to disable this setting again so the cache is maintained after restarts.
 
+In case you would like to setup the service via a text file, create a new file in `$OPENHAB_ROOT/conf/services` named `googletts.cfg`
+
+It's contents should look similar to:
```
Comment (jlaur) [inline bundles/org.openhab.voice.googletts/README.md:55]: Typo: "Its"

**Review by jlaur — CHANGES_REQUESTED** [review]
Looks good, only a small typo.

File: bundles/org.openhab.voice.googletts/README.md (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.voice.googletts/README.md:55]: Zut, it is wrong in several places.

Comment (lolodomo) [reply bundles/org.openhab.voice.googletts/README.md]: Fixed

Comment (jlaur) [reply bundles/org.openhab.voice.googletts/README.md]: Had to Google that... visiting Paris in April, so thanks. :-)

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**